src/server/routes/core.ts

```diff
@@ -20,4 +22,12 @@ router.get('/healthcheck', (_, res: Response) => {
   res.status(200).send('200 OK');
 });
 
+router.use('/maintenance', (_, res: ResponseWithRequestState) => {
```

Comment:
I think the core routes include cache headers

Can you add this to uncached routes instead?
https://github.com/guardian/gateway/blob/main/src/server/routes/index.ts#L22

Reply:
Refactored to add to uncached routes